packages/xrpl/src/models/ledger/Credential.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseLedgerEntry, HasPreviousTxnID } from './BaseLedgerEntry'
```

Comment:
Naming conventions in this repo typically have the first letter being lowercase (unless there's an acronym)

Reply:
Are you referring to the fields inside of the credential interface / fields inside of the transactions? I'm seeing capitalized fields for all of those, which is what I based it off of

Reply:
The file name

Reply:
Actually nvm, I see I'm wrong in this folder